Source: bug report here "https://sourceforge.net/tracker/?func=detail&aid=3529938&group_id=180858&atid=894869"
Upstream: reported to upstream
Reason: fix linking with new glib

--- configure.ac.org	2012-05-24 19:04:08.039732838 +0200
+++ configure.ac	2012-05-24 19:05:01.806838993 +0200
@@ -59,7 +59,7 @@
 
 if test "x$enable_gtk3" = "xyes" ; then
   CFLAGS="$CFLAGS -DENABLE_GTK3"
-  gtk_modules="gtk+-3.0 >= 3.0.0"
+  gtk_modules="gtk+-3.0 >= 3.0.0 gthread-2.0"
 else
   gtk_modules="gtk+-2.0 >= 2.12.0 gthread-2.0"
 fi
